sdb/commands/zfs/btree.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Btree(sdb.Walker):
+    """ walk  zfs_btree """
+
+    names = ["zfs_btree"]
+    input_type = "zfs_btree_t *"
+
+    def __init__(self, prog: drgn.Program, **kwargs):
```

Comment:
Can we add the type annotation for the return value.. i.e. `-> None:`

Can you also add any information about the benefits/drawbacks of using `**kwargs` here, vs. how the other commands define the constructor? Should we (later) adopt using `**kwargs` for all other commands?

Reply:
> Can you also add any information about the benefits/drawbacks ...
To be clear (after re-reading this, I think it might not be), I just mean as a comment/reply in the PR.. not something to add in the code.. I'm just curious if we should migrate to using `**kwargs` across the board.

Reply:
In this case, I don't think the constructor (`__init__`) is needed at all.

Reply:
@prakashsurya - I don't have a super strong intuition one way or the other - I was basing this on the implementation of `__init_subclass__` in `command.py`. I think having it match the other commands with `args: str = ""` makes sense to me. I think that's more explicit about what exactly the function has to work with. 

@ahrens - I added it to have a place to specify instance variables, but I guess since this is python, we don't actually need to do that.. Do you know if there's a stylistic imperative one way or the other? I guess I'm leaning towards getting rid of it, since we're not actually using it to instantiate `elem_size`.

Reply:
`__init__subclass__` is black magic.  You could model your `__init__` on other `__init__` methods, preferably of sibling classes (i.e. other subclasses of Walker).

AFAIK there isn't a convention to "declare" instance variables.  IMO it would be nice, though!

Reply:
Yeah that all makes sense to me. Interestingly, I just got this `pylint` warning:
```
sdb/commands/zfs/btree.py:79:8: W0201: Attribute 'elem_size' defined outside __init__ (attribute-defined-outside-init)
```